src/core/version.js

```diff
@@ -1,85 +1,5 @@
 // @flow
 
-import deepFreeze from "deep-freeze";
-export type VersionInfo = {|
-  +major: number,
-  +minor: number,
-  +patch: number,
-  +gitState: GitState,
-  +environment: Environment,
-|};
-export type GitState = {|
-  +commitHash: string,
-  +commitTimestamp: string, // YYYYmmdd-HHMM, in commit-local time
-  +dirty: boolean, // does the worktree have unstaged/uncommitted changes?
-|};
-export type Environment = "development" | "production" | "test";
+const packageJSON = require("../../package.json");
```

Comment:
nit: `packageJson`, please; we use consistent camel-case except where
there’s a strong API reason not to (e.g., `toJSON` integration point).
Cf. `JsonLog`.